lib/websocket_proxy.rb

```diff
@@ -15,7 +15,7 @@ def initialize(env, console)
       # Hijack the socket from the Rack middleware
       @ws = env['rack.hijack'].call
       # Set up the socket client for the proxy
-      @sock = TCPSocket.open(@console.host_name, @console.port)
+      Timeout.timeout(30) { @sock = TCPSocket.open(@console.host_name, @console.port) }
```

Comment:
@jrafanie this is a different issue and just a temporary solution, I'll explain tomorrow.